rdrf/registry/patients/admin_forms.py

```diff
@@ -344,16 +344,12 @@ def clinician_display_str(obj):
 
         # registered_clinicians field should only be visible for registries which
         # support linking of patient to an "owning" clinician
-        wgs_set_by_clinicians = False
         if self.registry_model:
             if not self.registry_model.has_feature(RegistryFeatures.CLINICIANS_HAVE_PATIENTS):
                 self.fields["registered_clinicians"].widget = forms.HiddenInput()
             elif instance and instance.registered_clinicians.exists():
-                wgs_set_by_clinicians = True
                 clinician_wgs = set([wg for c in instance.registered_clinicians.all() for wg in c.working_groups.all()])
-                self.fields["working_groups"].disabled = True
```

Comment:
I think we  should still keep a similar functionality: If clinicians are selected the associated working groups must be mandatory and only extra working groups should be selected/unselected at will.

Reply:
We can't disable the field because as I found out this week, some patients may want to share their clinical data with the clinicians in a working group, without necessarily registering any of them as a clinician.

So to make it mandatory, we could verify that the clinicians are selected in `clean_working_groups`, but I think it's simpler for the user to not have to select them

Reply:
Ok. It's fine then.